README.md

```diff
@@ -1,6 +1,6 @@
 # Lyo Validation library
 
-The library is intended to check whether resource instances conform to SHACL shapes.
+The library validates RDF resources against constraints defined either as [`SHACL`](https://www.w3.org/TR/shacl/) or [`OSLC Resource Shapes`](http://docs.oasis-open.org/oslc-core/oslc-core/v3.0/cs01/part6-resource-shape/oslc-core-v3.0-cs01-part6-resource-shape.html).
```

Comment:
this code was already in Jad's patch?!

Reply:
plus the `code` backticks are not really necessary

Reply:
Yes, it's from Jad's patch. 
@jadelkhoury